TenantFile/Api/Controllers/SmsController.cs

```diff
@@ -129,12 +129,20 @@ private async Task<IEnumerable<(string image, string thumbnail)>> SaveMedia(int
                 var httpClient = new HttpClient();
                 var response = await httpClient.GetAsync(mediaUrl);
                 var inputStream = await response.Content.ReadAsStreamAsync();
+                
                 using var image = SixLabors.ImageSharp.Image.Load(inputStream);
                 image.Mutate(x => x
-                     .Resize(0, 100));
+                     //.DetectEdges()                
+                     .Resize(new ResizeOptions()
+                        { 
+                           Mode = ResizeMode.Crop,
+                           Size = new Size(125, 100)
+                           
+                        }));
+                
                 var outputStream = new MemoryStream();
-                image.Save(outputStream, new PngEncoder());
-
+                image.Save(outputStream, encoder: new PngEncoder() { CompressionLevel = PngCompressionLevel.BestSpeed });
+                image.Dispose();
```

Comment:
Dispose isn't necessary, we're using a `using` statement on line 133 that will automatically clean up the resource.

Reply:
Removed. I'm not used to not having the brackets with the cope of the using statement. =)